data-serving/data-service/api/dataserver.api.yaml

```diff
@@ -0,0 +1,54 @@
+openapi: 3.0.3
```

Comment:
Glad you chose YAML over JSON. All of those defenseless curly braces...

Reply:
{
  {
    thanks!
  }
}